src/server/api.rs

```diff
@@ -0,0 +1,102 @@
+use std::{net::ToSocketAddrs};
+use tracing::info;
+use anyhow::Result;
+
+use crate::config;
+use tonic::{transport::Server, Request, Response, Status, Streaming};
+
+mod transaction {
+    tonic::include_proto!("transaction");
+}
+mod account {
+    tonic::include_proto!("account");
+}
+mod trees {
+    tonic::include_proto!("trees");
+}
+mod note {
+    tonic::include_proto!("note");
+}
+mod block {
+    tonic::include_proto!("block");
+}
+mod digest {
+    tonic::include_proto!("digest");
+}
+mod rpc {
+    tonic::include_proto!("rpc");
+}
```

Comment:
the generated code will load from `super::super::<mod>`, so these definitions are needed.